test/blackbox-tests/test-cases/cinaps/multiple-cinaps.t

```diff
@@ -24,6 +24,7 @@ Multiple cinaps stanzas in the same dune file
   _build/default/.cinaps.a7811055/cinaps.ml-gen:
   - dune:1
   - dune:2
+  -> required by Loading rules for directory _build/default/.cinaps.a7811055
```

Comment:
`Loading` shouldn't be capitalised as it is in the middle of a sentence.

Reply:
removed loading